themes/bene_child/Rakefile

```diff
@@ -11,7 +11,7 @@ end
 # Change basetheme.dev to your site path
 desc 'Running Browsersync'
 task :browsersync do
-  system 'browser-sync start --proxy "https://web.bene-test-1.localhost/hi-world" --files "css/*.css" --no-inject-changes'
+  system 'browser-sync start --proxy "web.jefe.localhost" --files "css/*.css" --no-inject-changes'
```

Comment:
Hate having this local URL business in here. Any way to work around this? Regardless, it seems like we should have it be something like "your-bene-site.localhost".